chia/util/streamable.py

```diff
@@ -116,7 +134,22 @@ def dataclass_from_dict(klass, d):
         return klass(d)
 
 
-def recurse_jsonify(d):
+@overload
+def recurse_jsonify(d: List[Any]) -> List[Any]:
+    ...
+
+
+@overload
+def recurse_jsonify(d: Tuple[Any, ...]) -> Tuple[Any, ...]:
+    ...
```

Comment:
Either a list or a tuple in results in a list out.  I feel like we ought to be able to do something about the `Any` here, but I think the `Tuple` -> `List` conversion will end up making a mess.  Though I'm not entirely sure why it is needed since in the end `json` will turn tuples into json arrays anyways.  And this all seems like a reimplementation of [JSONEncoder](https://docs.python.org/3.10/library/json.html#json.JSONEncoder).  Oh well, we'll get the easy thing anyways.

```suggestion
@overload
def recurse_jsonify(d: Union[List[Any], Tuple[Any, ...]) -> List[Any]:
    ...
```

Reply:
Added 6c18bfef66c2c8114f006a0ee0f1d2c3b3994b00, the suggestion was missing a `]`